addons/l10n_ec_ats/pestana_sri/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- encoding: utf-8 -*-
+
+from . import pestana_sri

```

Comment:
There is no need for a file or a folder just for a few fields, this modifications should be on your res_partner.py file on your models folder.